docs/backend/server_arguments.md

```diff
@@ -1,198 +1,236 @@
+| `sampling_backend` | Specifies the backend used for sampling. | None |
 
 ## Constrained Decoding
 
-* `grammar_backend`: The grammar backend for constraint decoding. Detailed usage can be found in this [document](https://docs.sglang.ai/backend/structured_outputs.html).
-* `constrained_json_whitespace_pattern`: Use with `Outlines` grammar backend to allow JSON with syntatic newlines, tabs or multiple spaces. Details can be found [here](https://dottxt-ai.github.io/outlines/latest/reference/generation/json/#using-pydantic).
+| Arguments | Description | Defaults |
+|----------|-------------| ----------|
+| `grammar_backend` | The grammar backend for constraint decoding. See [detailed usage](https://docs.sglang.ai/backend/structured_outputs.html). | None |
+| `constrained_json_whitespace_pattern` | Use with `Outlines` grammar backend to allow JSON with syntactic newlines, tabs, or multiple spaces. See [details](https://dottxt-ai.github.io/outlines/latest/reference/generation/json/#using-pydantic). |
 
 ## Speculative decoding
 
-* `speculative_draft_model_path`: The draft model path for speculative decoding.
-* `speculative_algorithm`: The algorithm for speculative decoding. Currently only [Eagle](https://arxiv.org/html/2406.16858v1) is supported. Note that the radix cache, chunked prefill, and overlap scheduler are disabled when using eagle speculative decoding.
-* `speculative_num_steps`: How many draft passes we run before verifying.
-* `speculative_num_draft_tokens`: The number of tokens proposed in a draft.
-* `speculative_eagle_topk`: The number of top candidates we keep for verification at each step for [Eagle](https://arxiv.org/html/2406.16858v1).
-* `speculative_token_map`: Optional, the path to the high frequency token list of [FR-Spec](https://arxiv.org/html/2502.14856v1), used for accelerating [Eagle](https://arxiv.org/html/2406.16858v1).
-
+| Arguments | Description | Defaults |
+|----------|-------------|---------|
+| `speculative_draft_model_path` | The draft model path for speculative decoding. | None |
+| `speculative_algorithm` | The algorithm for speculative decoding. Currently [EAGLE](https://arxiv.org/html/2406.16858v1) and [EAGLE3](https://arxiv.org/pdf/2503.01840) are supported. Note that the radix cache, chunked prefill, and overlap scheduler are disabled when using eagle speculative decoding. | None |
+| `speculative_num_steps` | How many draft passes we run before verifying. | None |
+| `speculative_num_draft_tokens` | The number of tokens proposed in a draft. | None |
+| `speculative_eagle_topk` | The number of top candidates we keep for verification at each step for [Eagle](https://arxiv.org/html/2406.16858v1). | None |
+| `speculative_token_map` | Optional, the path to the high frequency token list of [FR-Spec](https://arxiv.org/html/2502.14856v1), used for accelerating [Eagle](https://arxiv.org/html/2406.16858v1). | None |
 
 ## Double Sparsity
```

Comment:
drop this whole section. They are experimental and not useful.

Reply:
In fact, I can find it from [line 143 in server_args.py](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/server_args.py#L143) and no any comments there, which shows it's outdated.

If we drop "Double Sparsity" from this doc, then it will be inconsistent with `server_args.py`. 

Drop is easy but may cause inconsistency. Could we consider it in another separated PR?